src/pages/fileCentricCart/cartController.js

```diff
@@ -9,20 +9,37 @@ import CartView from './cartView';
 const cartController = () => {
   const cart = getCart();
   const ids = cart.fileIds ? cart.fileIds : [];
-  const defaultSortDirection = cart.sortDirection === '' ? table.defaultSortDirection || 'asc' : cart.sortDirection;
+  const defaultSortDirection = cart.sortDirection === '' || !cart.sortDirection || cart.sortDirection === null ? table.defaultSortDirection || 'asc' : cart.sortDirection;
   const CART_QUERY = defaultSortDirection === 'desc' ? GET_MY_CART_DATA_QUERY_DESC : GET_MY_CART_DATA_QUERY;
-  const defaultSortCoulmnValue = cart.sortColumn === '' ? table.defaultSortField || '' : cart.sortColumn;
+  const defaultSortColumnValue = cart.sortColumn === '' || !cart.sortColumn || cart.sortColumn === null ? table.defaultSortField || '' : cart.sortColumn;
```

Comment:
This looks like lot of clumsy code can we refactor this.

Reply:
Make this way to make sure even if the user is using the page for the first time, we still have the default value for the filecentric table.